### PR TITLE
Tests e2e

### PR DESCRIPTION
**Pull request by JureZmrzlikar**
In order the e2e tests to work:

- EMAIL, PASSW and URL have to be set in the beginning of the /resdk/tests/functionsl/e2e_test.py file

- @dblenkus's PR fix-descriptorschema has to be accepted in resolwe.

- TOOLS_REMOTE HOST has to be set as described here:
    http://resdk.readthedocs.io/en/latest/pipelines.html
    (paragraph starting with "Resolwe allows to place the ad-hoc scripts ...")

    My current hack for local Resolwe instance is:
    * set envar TOOLS_REMOTE_HOST
        TOOLS_REMOTE_HOST=/home/jure/devel/resolwe/tools
    * set genialis_base setting to:
        RESOLWE_CUSTOM_TOOLS_PATHS = ['/home/jure/devel/resolwe/tools']

    * I am not sure weather this workaround means that tests could miss an error...

### CODE REVIEW

**Comment by mstajdohar** [comment]
Rebase to master.

**Comment by JureZmrzlikar** [comment]
Rebased.

File: resdk/tests/files/custom_process.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  output:
+    - name: sum
+      label: Sum result
+      type: basic:integer
+  run:
+    runtime: polyglot
+    bash: |
+      OUT=`sum.py {{number1}} {{number2}}`
+      echo $OUT
+      re-save sum $OUT
```
Comment (mstajdohar) [inline resdk/tests/files/custom_process.yaml]: Missing end of line.

Comment (mstajdohar) [inline resdk/tests/files/custom_process.yaml]: I fixed it.

**Pull request merged**